packages/node/src/integrations/http/index.ts

```diff
@@ -118,12 +121,21 @@ export const instrumentOtelHttp = generateInstrumentOnce<HttpInstrumentationConf
   return instrumentation;
 });
 
+/** Exported only for tests. */
+export function _shouldInstrumentSpans(options: HttpOptions, clientOptions: Partial<NodeClientOptions> = {}): boolean {
```

Comment:
l: Do we really need a helper function + test for this? Imo could just be a const with description. Makes it rather hard to read.

Reply:
I just wanted to double check the behavior is correct without adding integration tests etc for this 😅 I always find it easy for little bugs to sneak in in such places, e.g. if you have a wrong `!` somewhere everything can become wrong, which can be easy to miss when testing "happy paths" in integration tests only.